flatliners/prometheusendpoint.py

```diff
@@ -0,0 +1,51 @@
+from time import sleep, time
+from collections import defaultdict
+from .baseflatliner import BaseFlatliner
+
+from prometheus_client import Gauge, start_http_server
+
+# Set up logging
+import logging
+_LOGGER = logging.getLogger(__name__)
+
+class PrometheusEndpoint(BaseFlatliner):
+    def __init__(self):
+        super().__init__()
+        self.pruning_interval = 5*60 # seconds # This value should be close to the scraping interval for your prometheus
```

Comment:
good practice is to have the comments *above* the line.
Also could you be more precise what scraping interval? 

The prometheus that is reading the weirdness score?

Reply:
I don't think we need this any more, keeping it equal to the chunk_size should work.

Reply:
An I will try to add a better comment

Reply:
If you completely remove it, then we don't even need a comment :)